src/mmap_windows.rs

```diff
@@ -61,6 +61,34 @@ pub const INVALID_HANDLE_VALUE: RawHandle = (-1isize) as RawHandle;
 #[allow(dead_code)]
 pub const ERROR_INVALID_PARAMETER: i32 = 87;
 
+/// `MmapRange` represents a range of arguments required to create Mmap regions.
+#[derive(Clone, Debug)]
+pub struct MmapRange {
+    size: usize,
+    file: Option<FileOffset>,
+}
+
+impl MmapRange {
+    /// Creates instance of the range with `size`.
+    pub fn new(size: usize) -> Self {
+        Self::with_file(size, None)
```

Comment:
Is this calling `mmap(2)` under the hood? If `size` is `0`, it would return `EINVAL`

Reply:
No, this is just creating a structure with arguments.